Source/JavaScriptCore/parser/Lexer.cpp

```diff
@@ -990,10 +990,15 @@ template <bool shouldCreateIdentifier> ALWAYS_INLINE JSTokenType Lexer<Latin1Cha
     if (shouldCreateIdentifier || m_parsingBuiltinFunction) {
         std::span identifierSpan { identifierStart, static_cast<size_t>(currentSourcePtr() - identifierStart) };
         if (m_parsingBuiltinFunction && isBuiltinName) {
-            if (isWellKnownSymbol)
-                ident = &m_arena->makeIdentifier(m_vm, m_vm.propertyNames->builtinNames().lookUpWellKnownSymbol(identifierSpan));
-            else
-                ident = &m_arena->makeIdentifier(m_vm, m_vm.propertyNames->builtinNames().lookUpPrivateName(identifierSpan));
+            if (isWellKnownSymbol) {
+                auto* symbol = m_vm.propertyNames->builtinNames().lookUpWellKnownSymbol(identifierSpan);
+                ASSERT_WITH_MESSAGE(symbol, "Private well-known symbol @@%s not found", WTF::String::fromUTF8(identifierSpan).utf8().data());
```

Comment:
Is WTF::String::fromUTF8(identifierSpan).utf8().data() a UAF? Since the String is dead when you actually go to print?